app/src/main/java/io/particle/android/sdk/ui/DeviceListActivity.java

```diff
@@ -76,7 +78,7 @@
     };
 
     @OnCheckedChanged({R.id.photonFilter, R.id.electronFilter, R.id.coreFilter,
```

Comment:
If we add any more devices, we should consider switching this to a vertically scrolling list.  Side scrolling always feels like a pain, and it takes longer to find what you want.